docs/pages/workflows/get-started.mdx

```diff
@@ -0,0 +1,143 @@
+
+If you haven't already, you'll need to create a project and sync it with EAS:
+
+<Collapsible summary="Create a project and sync it with EAS">
+  You can create a new project with the following command:
+
+{' '}
+
+<Terminal cmd={['$ npx create-expo-app@latest']} />
+
+Once you've created the project, login with your account:
+
+{' '}
+
+<Terminal cmd={['$ npx eas-cli@latest login']} />
+
+Finally, link the project you have locally with EAS:
+
+  <Terminal cmd={['$ npx eas-cli@latest init']} />
+</Collapsible>
+
+Then, create a directory named **.eas/workflows** at the root of your project with a **.yaml** file inside of it. For example: **.eas/workflows/hello-world.yaml**.
+
+## Write a workflow
+
+Each workflow consists of three top-level elements:
+
+- `name`: defines the name of the workflow. For example: "Hello World!"
+- `on`: defines when this workflow should be triggered. For example: when pushing a new commit to a GitHub branch.
+- `jobs`: a sequence of jobs which can depend on and pass data between each other. For example: a job that runs a unit test followed by a job that builds your project into an app.
+
+Here's an example of a workflow that prints "Hello world!":
+
+```yaml .eas/workflows/hello-world.yaml
+name: Hello World
+
+on:
+  push:
+    branches: ['*']
+
+jobs:
+  Hello World:
+    steps:
+      - run: echo "Hello, World"
+```
+
+Here's another example that creates and submits an iOS build of a project on every push to every branch. This is similar to running `eas build --platform ios --profile production --auto-submit`:
+
+```yaml .eas/workflows/ios-build-and-submit.yaml
+name: Release iOS app
+
+on:
+  push:
+    branches: ['*']
+
+jobs:
+  build:
+    type: build
+    params:
+      platform: ios
+      profile: production
+  submit:
+    needs: [build]
+    type: submit
+    params:
+      build_id: ${{ needs.build.outputs.build_id }}
+```
+
+## Configure your project
+
+To enable workflows to run on events from GitHub, you'll need to install Expo's GitHub app and connect it to your project. Here's how:
+
+- Navigate to your project's [GitHub settings](https://expo.dev/accounts/%5Baccount%5D/projects/%5BprojectName%5D/github)
+- Follow the UI to install the GitHub app. Then search for the GitHub repository that matches the Expo project and connect it.
+
+## Run your workflow
+
+Once you have a workflow file and your project is connected to Expo's GitHub app, you can trigger your workflow by pushing a commit to your GitHub repository. For the workflow to run, you'll need to make sure the trigger (defined with `on`) you defined in your workflow is met.
+
+Alternatively, you can trigger a workflow manually by running the following command:
+
+<Terminal cmd={['$ npx eas-cli@latest workflow:run .eas/workflows/<your-workflow-file>.yaml']} />
+
+Once you do, you can see your workflow running on your project's [workflows page](https://expo.dev/accounts/[account]/projects/[projectName]/workflows).
+
+## Next steps
```

Comment:
```suggestion
## Building blocks
```

Let's call this section building blocks instead of next steps since they (the links below) can be viewed in any order.